test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/PackageItemLoaderTests.cs

```diff
@@ -122,6 +122,9 @@ public async Task EmitsSearchTelemetryEvents()
             Assert.Equal(operationId, page0["ParentId"]);
             Assert.IsType<int>(page0["ResultCount"]);
             Assert.IsType<double>(page0["Duration"]);
+            Assert.IsType<double>(page0["AggregationDuration"]);
+            Assert.IsType<string>(page0["SourceTimingDuration"]);
+            Assert.Equal(1, ((string)page0["SourceTimingDuration"]).Split(',').Select(e => double.Parse(e)).Count());
```

Comment:
Are you going to emit a JSON number array?

Consider this helper method to validate the data:

```c#
      private void AssertJsonNumberArray(string text)
        {
            var dtc = new System.Runtime.Serialization.Json.DataContractJsonSerializer(typeof(double[]));
            var stringBytes = System.Text.Encoding.ASCII.GetBytes(text);
            using (var stream = new System.IO.MemoryStream(stringBytes))
            {
                var arr = (double[])dtc.ReadObject(stream);
                Assert.IsType(typeof(double[]), arr);
                Assert.Equal(1, arr.Length);
            }
        }
```

Reply:
If `SourceTimingDuraction` is a JSON array, how does this code work? The first and last elements should have `[` and `]` characters that break the parser, no? I'm confused.

Reply:
The test probably failed :D